web/src/hash_parser.ts

```diff
@@ -34,6 +34,18 @@ export function get_current_hash_section(): string {
     return get_hash_section(window.location.hash);
 }
 
+export function is_same_server_message(link: string): boolean {
```

Comment:
`is_same_server_message_link` should be the function name, and the parameter should be caller `url`, I think.

Reply:
I'll change the function name, but I deliberately named the parameter link, not url, since what we pass to it for any same server link is not a complete url, since it's a relative link and starts with `#`. Any other link though is a complete URL.

Hope that makes sense!

Reply:
I see, `url_fragment` would be a good name for that; see https://en.wikipedia.org/wiki/URI_fragment.